.github/workflows/documentation.yml

```diff
@@ -0,0 +1,27 @@
+name: Documentation
+
+on:
+  push:
+    branches:
+      - master
+    tags: '*'
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: julia-actions/setup-julia@latest
+        with:
+          version: '1.7'
+      - name: Install dependencies
+        run: julia --project=docs/ -e '
+              using Pkg; 
+              Pkg.develop(PackageSpec(path=pwd())); 
+              Pkg.instantiate()'
+      - name: Build and deploy
+        env:
+          DOCUMENTER_KEY: ${{ secrets.DOCUMENTER_KEY }} # If authenticating with SSH deploy key
```

Comment:
We actually don't need a deploy key, we just have to set the right permissions for the GITHUB_TOKEN.
```suggestion
```